cmd/report.go

```diff
@@ -71,37 +79,42 @@ func getBenchReportCommand() *cobra.Command {
 				return cobraext.FlagParsingError(err, cobraext.ReportOutputFlagName)
 			}
 
+			reportOutputPath, err := cmd.Flags().GetString(cobraext.ReportOutputPathFlagName)
+			if err != nil {
+				return cobraext.FlagParsingError(err, cobraext.ReportOutputPathFlagName)
+			}
+
 			isFull, err := cmd.Flags().GetBool(cobraext.ReportFullFlagName)
 			if err != nil {
 				return cobraext.FlagParsingError(err, cobraext.ReportFullFlagName)
 			}
 
-			srcPath, err := cmd.Flags().GetString(cobraext.BenchReportSrcPathFlagName)
+			newPath, err := cmd.Flags().GetString(cobraext.BenchReportNewPathFlagName)
```

Comment:
Ok, it is good that we changed from `src`/`target` to `new`/`old` because I was thinking that src refered to the old benchmark and target to the new one :slightly_smiling_face: